projects/swimlane/ngx-ui/src/lib/components/dropzone/dropzone.component.ts

```diff
@@ -43,8 +43,12 @@ export class DropzoneComponent implements OnInit {
 
   ngOnInit(): void {
     if (this.acceptedFileFormats && this.acceptedFileFormats.length) {
-      this.acceptedFileFormatsTextDisplay =
```

Comment:
Is this something we can use Intl.ListFormat for?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat